playbooks/galaxy.yaml

```diff
@@ -24,6 +24,7 @@
           CONN_MAX_AGE: 0
           OPTIONS:
             sslmode: '{{ postgres_sslmode }}'
+            sslrootcert: "{{'/etc/pki/tls/certs/ca-bundle.crt' if (postgres_sslmode in ['verify-ca', 'verify-full'])}}"
```

Comment:
This doesn't work for downstream since we're still using ansible 2.9 but upstream uses ansible-core 2.15

With 2.9 and one liner if statement, you need to define a else statement

```console
the inline if-expression on line 1 evaluated to false and no else section was defined
```